README.md

```diff
@@ -1 +1,160 @@
+> In order to use the project files, Kindly follow these steps:
+
+## Steps
+- To clone the project select the desired directory in cmd and run: `git clone https://github.com/Becky449/react-capstone.git`
+- Install npm dependencies: `npm install`
+- To check the live preview: `npm run start`
+- For StyLint test, run: `npx stylelint "**/*.{css,scss}"`
+- For WebHint test, run: `npx hint .`
+- For EsLint test, run: `npx eslint .`
+
+## Prerequisites
+- Have basic level knowledge about HTML/CSS/JS/ES6/React/Redux
+- Have basic level knowledge about linters/Webpack
+- Have basic level knowledge about the working of website
+
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone https://github.com/Becky449/react-capstone.git
+```
+2. To access cloned directory run:
+```bash
+cd react-capstone
+
+
+
+### Install
+> To install linters, react and other project's dependencies run:
+```bash
+npm install
+```
+## Run
+```bash
+1. To run this project :
+npm start
+2. open the urs localhost:8080 on your browser
+
+```
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode for React.\
+
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+## Authors
+
+👤 **Rabecca Nabwire**
+
+[GitHub](https://github.com/Becky449) | [LinkedIn](https://www.linkedin.com/in/rabeccanabwire)
+
+
+## ⭐️ Show your support <a name="support"></a>
+Give a ⭐️ if you like this project!
+
+## 🙏 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank [@Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for the templates and insights 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 📝 License <a name="license"></a>
+
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this project is properly licensed as the license is currently broken. This is a core requirement. This [MIT](https://choosealicense.com/licenses/mit/) recommended URL could help. Make sure that the content of this license is personalized, for example, the year should be the year this project is built and you also add your full name.